zoo-css-challenge/style.css

```
@@ -191,5 +205,5 @@ footer {
 /*Programs*/
 
 .programs .card {
-  display: block;
+  display: inline-grid;
```

```suggestion
  display: inline;
```
We can use the `inline` value instead of `inline-grid` which means something else